.cargo/config

```diff
@@ -1,3 +1,13 @@
-[build]
-# Postgres symbols won't ve available until runtime
-rustflags = ["-C", "link-args=-Wl,-undefined,dynamic_lookup"]
+# Auto-generated by pgx. You may edit this, or delete it to have a new one created.
+
+[target.x86_64-unknown-linux-gnu]
```

Comment:
do we have to specify the target?  Is there no way to wildcard this or just have a generic `[build]` section?  It seems to be the same for each target?

Reply:
We do need to specify this as far as I can tell. :( I think it'd be worth me poking at it again to see if there is some other way.

Reply:
It makes me think about the recent PR we had for pgx that added free-bsd support.  This means we'd have to list it here too -- and in every pgx extension.  Seems like we'd be chasing ghosts as more platforms are supported.

Reply:
I absolutely agree, this was just the only way that worked when I was trying to spike it.